sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -146,6 +147,20 @@ object RapidsPluginUtils extends Logging {
     }
     // set driver timezone
     conf.set(RapidsConf.DRIVER_TIMEZONE.key, ZoneId.systemDefault().normalized().toString)
+
+    // If spark.rapids.sql.multiThreadedRead.numThreads is not set explicitly, or it is set
+    // explicitly as value 0, then we derive it from other settings. Otherwise, we keep the
+    // users' setting.
+    val num_threads_key = RapidsConf.MULTITHREAD_READ_NUM_THREADS.key;
+    if (!conf.contains(num_threads_key) || conf.get(num_threads_key).toInt == 0) {
```

Comment:
use RapidsConf.isConfExplicitlySet

Reply:
> use RapidsConf.isConfExplicitlySet

Here `conf` is not type of `RapidsConf`, but type of `SparkConf`, so we can call `isConfExplicitlySet`.

Reply:
ok